src/main/kotlin/org/codefreak/codefreak/entity/EvaluationStep.kt

```diff
@@ -32,6 +33,10 @@ class EvaluationStep(
   @Enumerated(EnumType.STRING)
   var result: EvaluationStepResult? = null
 
+  var queuedOn: Instant? = null
+
+  var finishedOn: Instant? = null
```

Comment:
I think these should be `queuedAt` and `finishedAt` analogous to `createdAt` etc. in other entities.